src/test/java/hudson/tasks/AntTest.java

```diff
@@ -123,6 +124,12 @@ private void verify() throws Exception {
         assertEquals(1,l.length);
         r.assertEqualBeans(l[0],new AntInstallation("myAnt","/tmp/foo", JenkinsRule.NO_PROPERTIES),"name,home");
 
+        // Verify that PATH+ANT is set.
```

Comment:
OK. Can I assume you are doing at least manual testing?

`pipeline-model-definition` could add a `test`-scoped dep on this and do a full integration test there, if you also add a test dep on `jenkins-test-harness-tools`.